src/components/map/RouteMap.svelte

```diff
@@ -54,6 +55,25 @@
 		}
 	}
 
+	function addArrowToPolyline() {
```

Comment:
I think this should be moved into `src/lib/googleMaps.js`, and take the polyline as a param.

Reply:
Done